lib/nostrum/struct/ws_state.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Nostrum.Struct.WSState do
+  @moduledoc """
+  Struct representing the current WS state.
+  """
+
+  defstruct [
```

Comment:
We should include a `t` type in this module, like is present [here](https://github.com/Kraigie/nostrum/blob/master/lib/nostrum/struct/user.ex#L67).